// try and catch exception in JS

try {
  throw "lol";
} catch (err) {
  // err specfifes that whatever error is coming store in the variable err
  console.log(err);
}
